app.js

```diff
@@ -0,0 +1,87 @@
+var mode = prompt("Input mode: (b)asic, (a)dvanced, (bmi) calculator, (t)rip planner");
+switch (mode) {
+  case 'b':
+    console.log("Basic");
+    var action = prompt("Input a basic sum with two integers, seperated by space, and using +, -, *, or /");
+    var calc = action.split(" ");
+    if (calc.length !=3){
+      throw 'InvalidInputError';
+    }
+    var operator = calc[1];
+    calc[0] = parseInt(calc[0]);
+    calc[2] = parseInt(calc[2]);
+    switch (operator) {
+      case '+':
+        alert(calc[0]+calc[2]);
+        break;
+      case '-':
+        alert(calc[0]-calc[2]);
+        break;
+      case '*':
+        alert(calc[0]*calc[2]);
+        break;
+      case '/':
+        alert(calc[0]/calc[2]);
+        break;
+      default:
+        throw 'InvalidInputError';
+    }
+    console.log(calc);
+    break;
```

Comment:
Should be a space here before the next switch statement to matches the formatting you use throughout - try to keep your spacing consistent.